sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala

```diff
@@ -196,10 +179,12 @@ object GpuParquetScan {
     val schemaHasTimestamps = readSchema.exists { field =>
       TrampolineUtil.dataTypeExistsRecursively(field.dataType, _.isInstanceOf[TimestampType])
     }
-    def isTsOrDate(dt: DataType) : Boolean = dt match {
+
+    def isTsOrDate(dt: DataType): Boolean = dt match {
       case TimestampType | DateType => true
```

Comment:
Do we need to worry about TimestampNTZType? I don't think we support it here so it is probably fine to just add a comment and file a follow on issue to look at what it would take to support it.

Reply:
Filed https://github.com/NVIDIA/spark-rapids/issues/9707.